sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md

```diff
@@ -7,7 +7,7 @@ This sample demonstrates how to analyze the sentiment in one or more text inputs
 To create a new `TextAnalyticsClient` to analyze the sentiment in a text input, you need a Text Analytics endpoint and credentials.  You can use the [DefaultAzureCredential][DefaultAzureCredential] to try a number of common authentication methods optimized for both running as a service and development.  In the sample below, however, you'll use a Text Analytics subscription key.  You can set `endpoint` and `subscriptionKey` based on an environment variable, a configuration setting, or any way that works for your application.
```

Comment:
For these, let's say you'll use a .. subscription key **credential**.  And let's add a short sentence saying something like "passing in a TextAnalyticsSubscriptionKeyCredential" allows you to update credentials without creating a new client.

Reply:
Ohh nice catch. I didn't realized this was here. 
I will modify